### PR TITLE
Added (some) barcode creation to library

### PR DESCRIPTION
**Pull request by ZachMyers3**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -8,30 +8,37 @@ Example use
...
+    height += image_height
+    l.origin(0, height)
+    l.write_text('Happy Troloween!', char_height=5, char_width=4, line_width=60,
+                 justification='C')
+    l.endorigin()
+
+    print(l.dumpZPL())
+    l.preview()
 
 This will display the following preview image, generated using the `Labelary API <http://labelary.com/>`_:
```
Comment (cod3monk) [inline README.rst:42]: The URL of the image and raw text will need to be updated to also contain the newly added barcode.

File: zpl/label.py [hunk 2]
```diff
@@ -179,6 +180,25 @@ def write_field_number(self, number, name=None, char_height=None, char_width=Non
                 "characters and spaces"
             self.code += '"%s"' % name
 
+    def write_bar_code(self, height, barcode_type, oritentation='N', check_digit='N',
```
Comment (cod3monk) [inline zpl/label.py]: I would personally prefer `write_barcode`, also for all variables containing `bar_code` (to be replaced with `barcode`)

Comment (ZachMyers3) [reply zpl/label.py]: Check the latest commit, I believe I've refactored everything. 

Thanks for the quick response!

Comment (cod3monk) [reply zpl/label.py]: The example code in the README.rst file needs to be refactored too :)

Comment (cod3monk) [reply zpl/label.py]: Nevermind

File: README.rst (hunk 1 above)
Comment (ZachMyers3) [reply README.rst:42]: Check latest commits, changed the raw ZPL and labelary call

**Comment by cod3monk** [comment]
Looks good. Thanks for the PR!

**Pull request merged**